remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/transport/triple/TripleClientInvoker.java

```diff
@@ -143,6 +135,126 @@ public SofaResponse invoke(SofaRequest sofaRequest, int timeout)
 
     }
 
+    private Map<String, Method> methodMap = new ConcurrentHashMap<>();
+
+    @Override
+    public ResponseFuture asyncInvoke(SofaRequest sofaRequest, int timeout) throws Exception {
+        SofaResponseCallback sofaResponseCallback = sofaRequest.getSofaResponseCallback();
+        TripleResponseFuture future = new TripleResponseFuture(sofaRequest, timeout);
+
+        if (!useGeneric) {
+            Method m = methodMap.get(sofaRequest.getMethodName());
+            if (m == null) {
+                synchronized (sofaRequest.getInterfaceName()) {
```

Comment:
Using string as the key to synchronized may cause some problems, only `string.intern()` can make sure you got the same string.  For this case, maybe use `this` will be better?

Reply:
fixed